shiny_app/modules/visualisations/trend_mod.R

```diff
@@ -449,9 +472,9 @@ trend_mod_server <- function(id, filtered_data, geo_selections, techdoc) {
                       marker = list(enabled = TRUE)
                       ) |>
         hc_plotOptions(series=list(animation=FALSE)) |>
-        hc_xAxis(title = "") |>
         hc_yAxis(gridLineWidth = 0) |> # remove gridlines 
         hc_yAxis(title = list(text = type_definition)) |>
+        hc_xAxis(categories = unique(trend_data()$trend_axis), title = "") |>
```

Comment:
just curious what this line of code is doing/why its needed?

Reply:
ah it removes the axis label which would otherwise say "trend_axis" i'm still not sure why you need to specify the category as 'unique' but i'm sure they'll be a reason

Reply:
the title = "" bit removes the trend axis title. The categories = unique(trend_data()$trend_axis) is the fix that's just been applied after Liz noticed that the x-axis labels and the trend line were going funky when two areas are plotted that don't both have the same number of years to plot.